src/System.Private.ServiceModel/src/System.Private.ServiceModel.csproj

```diff
@@ -1,8 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Build">
-  <PropertyGroup>
```

Comment:
You should consider adding the binclash logger (https://github.com/dotnet/corefx/commit/c604732f4a781bc931444641268aa9edd81310fd) to wcf repo to help find any conflicts.